CHANGELOG.rst

```diff
@@ -1,10 +1,19 @@
 Changelog
 =========
 
-0.6.0 (unreleased)
+1.0.0 (unreleased)
 ------------------
 
-- Nothing changed yet.
+**Breaking changes**
+
+- By default record attachments now contains a relative location to
```

Comment:
We certainly need to discuss this yes! Can we schedule that some time tomorrow afternoon? Or Thursday?